cogs/startup.py

```diff
@@ -67,10 +67,23 @@ async def on_ready(self) -> None:
                 self.bot.set_main_guild(main_guild)
 
         if not main_guild:
+            logger.info(
+                "Invite URL: %s",
+                generate_invite_url(
+                    str(self.bot.application_id),
+                    str(settings["DISCORD_GUILD_ID"]))
```

Comment:
If the signature of the `generate_invite_url` function is updated to only allow integers, the entered values need not be cast to strings here.